src/boot_ppc_start.S

```diff
@@ -123,12 +123,10 @@ Also see MPC8544ERM
 
 _reset:
 
-#ifdef DEBUG
```

Comment:
Thanks @jpbland1 for spotting this. It was on purpose for release testing. I will investigate impact of this.